Earthfile

```diff
@@ -240,14 +240,18 @@ COPY_SOURCE:
 
 rpm-build:
     FROM +init --base fedora:rawhide
```

Comment:
I'm assuming that this creates a clean environment based on Fedora rawhide and so we don't need to run `mock`.  Is that right?

Reply:
That is correct, as an additional `chroot` may be somewhat redundant. Other Mock features might be useful here, but not at the moment, as far as I am aware.